benches/parallel_specs.rs

```diff
@@ -36,9 +36,9 @@ fn build() -> Planner<()> {
     {
         let ents: Vec<Entity> = w.create_iter().take(N).collect();
 
-        let mut rs = w.write::<RComp>();
-        let mut w1s = w.write::<W1Comp>();
-        let mut w2s = w.write::<W2Comp>();
+        let mut rs = w.write::<RComp>().pass();
```

Comment:
Yes, but I don't think you'd see that difference given that the operation is done only once.